examples/jsm/nodes/accessors/LineMaterialNode..js

```diff
@@ -0,0 +1,43 @@
+import MaterialNode from './MaterialNode.js';
+import { addNodeClass } from '../core/Node.js';
+import { nodeImmutable } from '../shadernode/ShaderNode.js';
+
+class LineMaterialNode extends MaterialNode {
+
+	constructor( scope ) {
+
+		super();
+
+		this.scope = scope;
+
+	}
+
+
+	construct( /* builder */ ) {
+
+		const node = this.getFloat( this.scope );
+		this.node = node;
+
+		return node;
```

Comment:
The code below should work, or maybe we have some issue in the build process.

```js
class LineMaterialNode extends MaterialNode {

	constructor( scope ) {

		super();

		this.scope = scope;

	}


	construct( /* builder */ ) {

		return this.getFloat( this.scope );

	}

}
```

Reply:
I tried that initially, but it failed as described. I traced it through the path for varying()  where the reference is included in the vertex shader flow and the uniform name is omitted in the code produced. 

The output from the construct() phase is entered into the nodes properties[ 'fragment' ] so isn't available at that point. I don't understand the code well enough to known if there is a better solution.

Reply:
Hmm... Make sense, `construct()` must build the output node once. I will fix this.